internal/terraform/module/module_manager_test.go

```diff
@@ -408,3 +413,135 @@ func testLogger() *log.Logger {
 
 	return log.New(ioutil.Discard, "", 0)
 }
+
+func TestSchemaForVariables(t *testing.T) {
```

Comment:
Thanks for the test 👍🏻 

I assume most of it is already tested in https://github.com/hashicorp/terraform-schema/blob/main/schema/variable_schema_test.go though, isn't it?

Reply:
you're right, I removed all of the tests cases but one so that only the actual `SchemaForVariables()` is tested